sdk/communication/azure-communication-phonenumbers/README.md

```diff
@@ -143,6 +182,46 @@ poller = self.phone_number_client.begin_update_phone_number_capabilities(
 )
 ```
 
+### SipRoutingClient
+
+#### Retrieve SIP trunks and routes
+
+Get the list of currently configured trunks or routes.
+
+```python
+trunks = sip_routing_client.get_trunks()
+routes = sip_routing_client.get_routes()
```

Comment:
Ideally for the code snippets we would actually demonstrate some usage:
```python
trunks = client.get_trunks()
for trunk in trunks:
    print(trunk.sip_signaling_port)
```

Reply:
Added print of result properties.